### DISCUSSION TITLE
I can't create a new project

### DISCUSSION THREAD

**Discussion by nuxtscientist** [vercel/community#6659] [Help] [answered]
### Summary

When I want to create a new project, to create a repo on github, it asks me for a name that doesn't already exist and I enter a really new name. Then vercel just creates the repository and gives the error “The repository "nextjs-boilerplatea" is owned by an organization. Please link it to a Vercel team instead.” error.

### Steps to Reproduce

![image](https://github.com/vercel/community/assets/168138275/7be3cf20-fb54-41cd-81f6-3e06144962fa)

**Comment by versecafe** [comment, marked as answer]
@nuxtscientist You can't create projects for a repository owned by an organization on a hobby team, you'd need to use a pro team or make a fork to your personal github and deploy the fork.

**Comment by johnnykoo84** [comment]
I got into the same problem and problem solved. i think Vercel needs to provide better error message to the users.

**Reply by versecafe** [reply]
Adding a link to more in depth documentation on exactly what the issue is would probably help out a decent number of people, I think a decent number of the error messages could use some links to docs on common solutions and a in depth explanation.

[a comment by an automated account is not shown]
